ui/components/Navbar/Navbar.js

```diff
@@ -1,456 +1,456 @@
+            className={classes.rightMargin}
+            style={!isDrawerCollapsed ? { display: "none" } : { marginLeft: "4px" }}
+          >
+            <Tooltip title="Help" placement={isDrawerCollapsed ? "right" : "top"}>
+              <IconButton
+                className={isDrawerCollapsed ? classes.collapsedHelpButton : classes.rightTranslate}
+                onClick={toggleSpacing}
+              >
+                <HelpIcon className={classes.helpIcon} style={{ fontSize: "1.45rem" }} />
+              </IconButton>
+            </Tooltip>
+          </ListItem>
+        </ButtonGroup>
+      </div>
+    );
+  };
+
+  return (
+    <nav className={isDrawerCollapsed ? classes.drawerCollapsed : classes.drawer}>
+      <Drawer
+        variant={isMobile ? "temporary" : "permanent"}
+        open={isDrawerOpen}
+        onClose={() => setIsDrawerOpen(false)}
+        className={isDrawerCollapsed ? classes.sidebarCollapsed : classes.sidebarExpanded}
+        classes={{ paper: isDrawerCollapsed ? classes.sidebarCollapsed : classes.sidebarExpanded }}
+        sx={{ zIndex: theme.zIndex.drawer + 10, position: "relative", width: "inherit" }}
+      >
+        <HiddenscrollbarStyle>
+          <List disablePadding>
+            <div className={classname}>
+              <ArrowBackIosIcon
+                style={{ verticalAlign: "middle", margin: "0.7rem -0.1rem 0.7rem 0.3rem", fontSize: "0.9rem" }}
+                onClick={() => toggleMiniDrawer()}
+                alt="Sidebar collapse toggle icon"
+              />
+            </div>
+            <ListItem
+              component="a"
+              onClick={handleTitleClick}
+              className={classNames(classes.firebase, classes.item, classes.itemCategory, classes.cursorPointer)}
+            >
+              <Avatar
+                className={isDrawerCollapsed ? classes.mainLogoCollapsed : classes.mainLogo}
+                src="/static/img/meshery-logo.png"
+              />
+              <Avatar
+                className={isDrawerCollapsed ? classes.mainLogoTextCollapsed : classes.mainLogoText}
+                src="/static/img/meshery-logo-text.png"
+              />
+            </ListItem>
+            {renderNavItems(categories)}
+            {extensionsNavigator && extensionsNavigator.length ? (
+              <React.Fragment>
+                <Divider className={classes.divider} />
+                {renderNavigatorExtensions(extensionsNavigator, 1)}
+              </React.Fragment>
+            ) : null}
+            <Divider className={classes.divider} />
+          </List>
+        </HiddenscrollbarStyle>
+        {renderExternalLinkItems()}
+        {
+          <MesheryServerVersionContainer>
+            {({ serverVersion }) => (
+              <div>
+                {isDrawerCollapsed ? (
+                  <div style={{ color: "white", fontSize: "0.7rem", textAlign: "center", paddingBottom: "0.5rem" }}>
+                    {serverVersion.build}{" "}

```

Comment:
The `build` property of the `serverVersion` object has to be rendered.

Reply:
Oh , i can't beleive i missed that!! 😅

Reply:
nice catch @abhinavkrin.